### DISCUSSION TITLE
How to use WordPress Playground for handovers

### DISCUSSION THREAD

**Discussion by ironnysh** [WordPress/developer-blog-content#245] [Topic Ideas]
After writing [an overview of what's possible with Playground](https://github.com/WordPress/developer-blog-content/issues/198) for different audiences, I'd like to share a practical example focused on a use case relevant for developers.

The tutorial demonstrates how to work with the JavaScript API to create a [live WordPress website](https://github.com/ironnysh/playground-demo-handover) developers can share with clients. The project includes:
- A modified theme
- A custom plugin, and
- A snippet that generates a new post (complete with the block markup).

The text is already written :-) [See the draft here](https://docs.google.com/document/d/1c31vE7U9eSLfHc9bE1-6E3BTK6bB2s6OhTfLbO_l7zQ/edit?usp=sharing).

**Comment by bph** [comment]
Topic was approved. the conversation moved to the issue 

- #247

**Comment by bph** [comment]
Published: [How to use WordPress Playground for interactive demos](https://developer.wordpress.org/news/2024/04/25/how-to-use-wordpress-playground-for-interactive-demos/)
